apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/RuntimeTestsApi.ts

```diff
@@ -158,8 +133,9 @@ export async function wait(delay: number) {
   return testRunner.wait(delay);
 }
 
+// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
Because we copy class method, so the state is not preserved. We can 

- disable eslint
- change notify function into arrow function
- change notify into a static function and provide option to ignore static unboud functions https://typescript-eslint.io/rules/unbound-method/#ignorestatic in eslint config

Personally I suggest using ignorestatic, because static functions don't rely on class scope and should be safe to use.